README.md

```diff
@@ -17,98 +17,69 @@ The [Blazor DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagr
 
-[Documentation](https://blazor.syncfusion.com/documentation/datagrid/getting-started/?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples)
+[Online Examples](https://blazor.syncfusion.com/demos/datagrid/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=blazor-datagrid-github-samples)
 
-[Suggest a Feature](https://www.syncfusion.com/feedback/blazor-components?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples)
+[Knowledge base](https://support.syncfusion.com/kb/article/9909/how-to-get-started-easily-with-syncfusion-blazor-table?utm_source=github&utm_medium=listing&utm_campaign=blazor-datagrid-github-samples)
 
 ## About Syncfusion Blazor Components
 
-We have other popular [Blazor Components](https://www.syncfusion.com/blazor-components?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples) such as [Charts](https://www.syncfusion.com/blazor-components/blazor-charts?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [Scheduler](https://www.syncfusion.com/blazor-components/blazor-scheduler?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [Diagram](https://www.syncfusion.com/blazor-components/blazor-diagram?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [Word Processor](https://www.syncfusion.com/blazor-components/blazor-word-processor?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), and more.
+Syncfusion's [Blazor components](https://www.syncfusion.com/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-datagrid-github-samples) library offers over 80 UI components to work with Blazor server-side and client-side (Blazor WebAssembly) projects seamlessly. In addition to DataGrid, we provide popular Blazor components such as [Charts](https://www.syncfusion.com/blazor-components/blazor-charts?utm_source=github&utm_medium=listing&utm_campaign=blazor-datagrid-github-samples), 
+[Scheduler](https://www.syncfusion.com/blazor-components/blazor-scheduler?utm_source=github&utm_medium=listing&utm_campaign=blazor-datagrid-github-samples), [Diagram](https://www.syncfusion.com/blazor-components/blazor-diagram?utm_source=github&utm_medium=listing&utm_campaign=blazor-datagrid-github-samples), and [Word Processor](https://www.syncfusion.com/blazor-components/blazor-word-processor?utm_source=github&utm_medium=listing&utm_campaign=blazor-datagrid-github-samples).
 
+### About Syncfusion
 
-## About Syncfusion
+Founded in 2001 and headquartered in Research Triangle Park, N.C., Syncfusion has more than 29,000+ customers and more than 1 million users, including large financial institutions, Fortune 500 companies, and global IT consultancies.
```

Comment:
remove +
like - more than 29,000 customers and more than 1 million users..............

Reply:
Removed